prac_05/state_names.py

```diff
@@ -0,0 +1,25 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+# TO DO: Reformat this file so the dictionary code follows PEP 8 convention
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory",
+                "WA": "Western Australia", "ACT": "Australian Capital Territory",
+                "VIC": "Victoria", "TAS": "Tasmania"}
+MENU = """Type the state code to reveal its name
+To list all codes and names press enter anytime."""
+# print(CODE_TO_NAME)
+print(MENU)
+state_code = input("Enter short state: ").upper()
+while state_code != "":
+    if state_code in CODE_TO_NAME:
+        print(state_code, "is", CODE_TO_NAME[state_code])
```

Comment:
Keep consistent print format (below you use the .format method)
```suggestion
        print("{} is {}".format(state_code, CODE_TO_NAME[state_code]))
```

Reply:
Wasn't instructed to edit that part of the copied source code. I get your point though.